src/main/java/racingcar/domain/CarStateInRace.java

```diff
@@ -0,0 +1,29 @@
+package racingcar.domain;
+
+import racingcar.domain.Car;
+
+public class CarStateInRace {
+
+    private final static String STEP = "-";
+    private final static int ZERO = 0;
+
+    private String currentCarStatement;
+    private final Car car;
+    private StringBuilder stringBuilder = new StringBuilder();
+
+    public CarStateInRace(final Car car) {
+        this.car = car;
+    }
+
+    public void convertCurrentCarStatement() {
+        stringBuilder.append(car.getName() + ": ");
```

Comment:
```suggestion
        stringBuilder
          .append(car.getName())
          .append(": ");
```
가독성을 위해 수정해보는 것은 어떨까요?